modules/VtigerBackup/VtigerBackup.php

```diff
@@ -26,7 +26,12 @@
  * @author MAK
  */
 class VtigerBackup {
-	private $folderList = array('storage','test','user_privileges');
+	// 'storage' folder removed from backup, the CBX one is too large and will run into memory limits
```

Comment:
Yes this was only meant for my personal fork, not as a suggestion for coreBOS. I'm not familiar with the external backup though.. Will research